apps/passport-client/components/screens/LoginScreens/OneClickLoginScreen.tsx

```diff
@@ -26,18 +38,50 @@ export function OneClickLoginScreen(): JSX.Element | null {
     }
   }, [targetFolder]);
 
+  const handleLoadTicketPreviews = useCallback(async () => {
+    if (!email || !code) {
+      return;
+    }
+    try {
+      const previewRes = await requestGenericIssuanceTicketPreviews(
+        serverUrl ?? appConfig.zupassServer,
+        email,
+        code,
+        pipelineId
+      );
+
+      setLoading(false);
+      if (previewRes.success) {
+        setTicketPreviews(previewRes.value.tickets);
+      } else {
+        setError(previewRes.error);
+      }
+    } catch (err) {
+      await dispatch({
```

Comment:
related to previous comment about handling the 400 -- should this just actually call `handleOneClickLogin()` e.g. for the Esmeralda and Prague links that were sent out?